src/main/scala/beam/sim/config/BeamConfig.scala

```diff
@@ -37,6 +39,7 @@ object BeamConfig {
   object Beam {
 
     case class Agentsim(
+      randomSeed: scala.Int,
       agentSampleSizeAsFractionOfPopulation: scala.Double,
```

Comment:
@zneedell randomSeed was never checked in for beam-template.conf. This will cause BeamConfig to be overwritten in future PRs

Reply:
For posterity - this was resolved in https://github.com/LBNL-UCB-STI/beam/pull/3520